README.md

```diff
@@ -29,9 +29,35 @@ I did this ages ago, so I don't really remember.
 
 1.  Install `gnome-common`
 2.  Install the `python3-arrow`  and `python3-markdown`
-3.  Install a SCSS compiler (eg. `rubygem-sass` on Fedora).  This is very
+3.  Install `libglib2.0-dev`,`libgtk-3-dev`,`libsoup2.4-dev`,`libjson-glib-dev`
+4.  Install a SCSS compiler (eg. `rubygem-sass` on Fedora).  This is very
     important because otherwise it will fail to build.
-
+    For Ubuntu run the following commands :
+    `sudo apt-get install rubygems ruby-dev`
+    `sudo su -c "gem install sass"`
+    `sudo gem install sass`
+5.  Install the package `gobject_introspection` by using the command `sudo apt install gobject-introspection`
+
+Installing libglib2.0_50
```

Comment:
Wow, this is a hard way to do it.  Right now, I'm just using GNOME's `jhbuild` tool to install everything.  It might help you too!